robusta_krr/main.py

```diff
@@ -138,15 +138,16 @@ def run_strategy(
                 ),
                 prometheus_cluster_label: Optional[str] = typer.Option(
                     None,
-                    "--prometheus-cluster-label",
+                    "--prometheus-cluster-value",
```

Comment:
same

Reply:
Yes, this one I did change but if I understand the runner code correctly, it uses `-l` which is still supported and equivalent to this flag. So no change needed.